src/main.ts

```diff
@@ -37,9 +37,7 @@ window.requestAnimFrame = (() => {
     window.mozRequestAnimationFrame ||
     window.oRequestAnimationFrame ||
     window.msRequestAnimationFrame ||
-    function (callback: () => void) {
-      window.setTimeout(callback, 1000 / 60);
-    };
+    ((callback) => window.setTimeout(callback, 1000 / 60));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 60](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)